Lesson_11/test_logger.py

```diff
@@ -0,0 +1,39 @@
+# test_logger.py
+import unittest
+from unittest.mock import patch
+import logging
+from logger import log_event
+
+
+class TestLogEvent(unittest.TestCase):
```

Comment:
Please, better use pytest library for testing prospective. Unittest is not as good framework for automation testing.